Cargo.toml

```diff
@@ -20,7 +20,8 @@ rand_core = "0.6"
 rand = "0.8"
 rand_chacha = "0.3.1"
 tikv-jemallocator = "0.5.4"
-wsts = { version = "8.1", default-features = false }
+# wsts = { version = "8.1", default-features = false }
+wsts = { path = "/Users/aaron/devel/wsts" }
```

Comment:
This was necessary to get access to some private variables in the coordinator. If we want to continue reusing the fire coordinator implementation in the miner's coordinator, I can open a PR with the necessary changes.